static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-moment.js

```diff
@@ -81,13 +81,11 @@ export const acquisitionsBannerMomentTemplate = (
 
             <div class="moment-banner__buttons">
                 <div class="engagement-banner__cta">
-                    <a tabindex="3" class="button  engagement-banner__button  engagement-banner__button--moment-link" href="${
-                        params.linkUrl
-                    }">
+                    <a tabindex="3" class="button  engagement-banner__button  engagement-banner__button--moment-link" href="https://www.theguardian.com/p/cfm63">
```

Comment:
Is this URL supposed to exist already? I get a 404 when I try to access it.

Reply:
it's the short url for an article that has not yet been published. The banner will not be switched on until it has

Reply:
No, the link will go live when the Moment is launched.

Reply:
👍